summary_covidseq.R

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env Rscript
+# Author: OLP/2022
+# Usage: 1. summary for PatientSamples on the SampleSheet after the analysis phase is completed
+#        2. Arguments: run name
+#        3. Example: Rscript / . . . ./summary_v2.R UT-A01290-220416
+
+ 
+# libraries
+library("easypackages")
+libraries("tidyverse","XML","xml2","seqinr","lubridate"); 
+
+
+args = commandArgs(trailingOnly=T)
+date<-ymd(substr(args[1], 11, 16));runPath <-paste('/Volumes/NGS/Analysis/covidseq',args[1],sep="/"); 
+head(date)
+runPath
+
+#=============================================
+# Type Of Sequencer : NovaSeq or NextSeq
+Sequencer <-substr(args[1], 4, 4); Sequencer; dat<-substr(args[1], 11, 16);dat
+if (Sequencer =="A"){ TypeOfSequencer<-"NovaSeq";Pt<-"A01290"} else {TypeOfSequencer<-"NextSeq";Pt<-"NB551133"};Pt; 
+t=.5;Sys.sleep(t)
+
+#=============================================
+# Path to Analysis
+ss1 = paste(runPath,"covidseq_output/SampleSheet_Intermediate.csv",sep ="/"); 
+ss2 = paste(runPath,"covidseq_output/lineage/combined_lineage_report.csv",sep ="/"); 
+ss3 = paste(runPath,"covidseq_output/Logs_Intermediates/VirusDetection/kmer/Summary.tsv",sep ="/");
+
+#=============================================
+df1 = read.csv(ss1); # SampleSheet
+df2 = read.csv(ss2); # combined_llineage_report
+df3 = read.table(file = ss3, sep = '\t', header = T); # amplicons 
+
+#=============================================
+# detecting the lineage tool version from the  main_seq xxxx  script
+s<-"/Volumes/NGS/Bioinformatics/dragen_scripts/main_seq_run_v2.sh"
+sx = read.csv(s)[(70:70),];
+Lineage_tools<- substring(sx, 7,40);
+print(Lineage_tools)
```

Comment:
This file no longer exists. I thought we switched to running pangolin directory from the docker container?